deepctr/layers/sequence.py

```diff
@@ -640,7 +640,8 @@ def build(self, input_shape):
         # Second part, apply the cosine to even columns and sin to odds.
         position_enc[:, 0::2] = np.sin(position_enc[:, 0::2])  # dim 2i
```

Comment:
can you help us solve this issue?  https://github.com/shenweichen/DeepCTR/issues/379

Reply:
modified. you can check it.